cmd/connect.go

```diff
@@ -74,6 +78,12 @@ var connectCommand = Command{
 			Help: "Allow one or more failures",
 			Flag: "true",
 		},
+		{
+			Name:  connectTerminalOption,
+			Short: "T",
+			Help:  "Start a terminal session",
```

Comment:
Should we mention that this option is exclusive (i.e. no port forwarding will happen when this is specified)? Alternatively, should this just be a different command? E.g. `qbee-cli shell <device-id>` would be nice IMO.